draft-ietf-quic-recovery.md

```diff
@@ -763,37 +770,6 @@ duration of the timer is based on the timer's mode, which is set in the packet
 and timer events further below.  The function SetLossDetectionTimer defined
 below shows how the single timer is set.
 
-#### Crypto Handshake Timer
-
-When a connection has unacknowledged crypto packets, the crypto handshake timer
-is set and when it expires, all unacknowledged CRYPTO data is retransmitted.
-
-When stateless rejects are in use, the connection is considered immediately
-closed once a reject is sent, so no timer is set to retransmit the reject.
-
-Version negotiation packets are always stateless, and MUST be sent once per
-handshake packet that uses an unsupported QUIC version, and MAY be sent in
-response to 0-RTT packets.
-
-#### Tail Loss Probe and Retransmission Timer
-
-Tail loss probes {{?TLP}} and retransmission timeouts {{?RFC6298}}
-are timer based mechanisms to recover from cases when there are
-outstanding retransmittable packets, but an acknowledgement has
-not been received in a timely manner.
-
-The TLP and RTO timers are armed when there are no unacknowledged crypto
-packets.  The TLP timer is set until the max number of TLP packets have been
-sent, and then the RTO timer is set.
```

Comment:
Did you mean to delete all of this?

Reply:
Yes, I believe it's duplicative of the more detailed text above.

Reply:
OK.  It seemed entirely unrelated, but I see we have text on both elsewhere.